### DISCUSSION TITLE
Decentralized Stable coin Error

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1838] [General]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/9e29b177-0ebf-47d2-95e9-aba95473df80)
I am getting this error following exact tutorial
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/7789e0d2-4bcc-4f88-ac8e-450168fc9aa3)

**Comment by urosognjenovic** [comment]
The ```Ownable``` contract has been changed. Now it initializes the initial owner in the constructor. Here it is:
```
constructor(address initialOwner) {
        if (initialOwner == address(0)) {
            revert OwnableInvalidOwner(address(0));
        }
        _transferOwnership(initialOwner);
    }
```

Change your code to this:
```
constructor(address initialOwner) ERC20("DecentralizedStableCoin", "ERC20") Ownable(initialOwner) {}
```
And make sure to pass the ```initialOwner``` variable to the constructor when creating a new DecentralizedStableCoin.

**Reply by panditdhamdhere** [reply]
Thanks ser! it is really helpful! It solved my problem

**Reply by urosognjenovic** [reply]
No problem! I would appreciate it if you could mark my reply as answer 🙏

**Reply by panditdhamdhere** [reply]
I am not getting this option here can you please share how can I mark this comment as a answer so it'll be helpful for everyone!

**Reply by urosognjenovic** [reply]
Clicking on the three dots in the top right corner of my reply should give you that option.

**Reply by panditdhamdhere** [reply]
@thebraveman17 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/4abad8b8-ed32-491c-81e1-caa623047fe4)
it showing this only

**Reply by urosognjenovic** [reply]
Never mind then. Thank you for your effort.

**Comment by panditdhamdhere** [comment]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/ea5868a2-f7eb-4dab-8ce0-5f9fd30e45eb)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/c8c5a42f-b423-4cd5-8d06-99b2f6350ed6)

Also what I can pass here as an Argument since you solved my last ownable problem! Patrick passes nothing and due to this my test failing
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/ca25eea2-5e39-437e-ae52-41dd39cf5db1)

Hi @thebraveman17 can you please help me with this
